### DISCUSSION TITLE
Access the database.db file

### DISCUSSION THREAD

**Discussion by Lime-Parallelogram** [filebrowser/filebrowser#1785] [Q&A]
Hello everyone,
Thank you for all your work on this piece of software - it is just what I need. I would however like to write a program that will allow me to add a load of users quickly. I figured I could probably do this using the database file but I can't seem to open it. Is it possible to do this? Do I need a decryption key from somewhere?

Many thanks!

**Comment by ramiresviana** [comment]
The command line interface allows you to manipulate the database, check the documentation at https://filebrowser.org/cli
